tableauserverclient/server/request_factory.py

```diff
@@ -260,14 +268,22 @@ def add_req(self, user_item):
 
 
 class WorkbookRequest(object):
-    def _generate_xml(self, workbook_item):
+    def _generate_xml(self, workbook_item, connection_credentials=None):
         xml_request = ET.Element('tsRequest')
         workbook_element = ET.SubElement(xml_request, 'workbook')
         workbook_element.attrib['name'] = workbook_item.name
         if workbook_item.show_tabs:
             workbook_element.attrib['showTabs'] = str(workbook_item.show_tabs).lower()
         project_element = ET.SubElement(workbook_element, 'project')
         project_element.attrib['id'] = workbook_item.project_id
+        if connection_credentials:
+            credentials_element = ET.SubElement(workbook_element, 'connectionCredentials')
+            credentials_element.attrib['name'] = connection_credentials.name
+            credentials_element.attrib['password'] = connection_credentials.password
+            if connection_credentials.embed:
+                credentials_element.attrib['embed'] = 'true'
+            else:
+                credentials_element.attrib['embed'] = 'false'
```

Comment:
I asked him to not do that because I don't like relying on the str value, what if someone passes in an overridden `True`? This makes it explicit what two values are allowed from reading the code